examples/research_projects/dreambooth_inpaint/train_dreambooth_inpaint.py

```diff
@@ -719,6 +763,12 @@ def collate_fn(examples):
                 progress_bar.update(1)
                 global_step += 1
 
+                if global_step % args.checkpointing_steps == 0:
+                    if accelerator.is_main_process:
+                        save_path = os.path.join(args.output_dir, f"checkpoint-{global_step}")
```

Comment:
naming is good for me, since there is no need to allow the user to overwrite "checkpoint-..." with something else

Reply:
Sorry for that I am newbie in github

Just want you to know that this line may have to be changed

examples/research_projects/dreambooth_inpaint/train_dreambooth_inpaint.py

675 for epoch in range(first_epoch, args.num_epochs):
==>
675 for epoch in range(first_epoch, args.num_train_epochs):